packages/browsers/README.md

```diff
@@ -19,7 +19,7 @@ npm i @puppeteer/browsers
 ```json
 {
   "scripts": {
-    "browsers": "@puppeteer/browsers --help"
+    "browsers": "npx @puppeteer/browsers --help"
```

Comment:
```suggestion
    "browsers": "npx @puppeteer/browsers"
```
Again if it's a command we are executing `--help` will stop you from doing `npm run browsers install chrome@latest`

Reply:
This was not a goal of this example but since it is confusing I am gonna remove all of it.